index.js

```diff
@@ -45,6 +46,20 @@ function handleArgs(cmd, args, opts) {
 	};
 }
 
+function handleInput(spawned, opts) {
+	var input = opts.input;
+
+	if (input === null || input === undefined) {
+		return;
+	}
+
+	if (isStream(input)) {
+		input.pipe(spawned.stdin);
+	} else {
+		spawned.stdin.end(input);
+	}
```

Comment:
@kevva Open an issue on https://github.com/sindresorhus/eslint-plugin-xo and we can discuss.